objects-revisited/main.js

```diff
@@ -0,0 +1,84 @@
+const business = {
+  opens: '9am',
+  closes: '5pm',
+  totalEmployees: 4,
+  daysOpen: ['M', 'T', 'W', 'Th', 'F'],
+  employees: {
+    john: {
+      position: 'Cashier',
+      daysOfWeekWorking: ['M', 'W', 'Th', 'F']
+    },
+    ted: {
+      position: 'Stocking',
+      daysOfWeekWorking: ['M', 'T', 'F']
+    },
+    jack: {
+      position: 'Manager',
+      daysOfWeekWorking: ['M', 'T', 'W', 'Th', 'F']
+    },
+    jasmine: {
+      position: 'Barista',
+      daysOfWeekWorking: ['M', 'W', 'Th']
+    },
+  }
+}
+
+function addWeekends() {
+  business.daysOpen.push('S');
+  business.daysOpen.push('Su');
+  for (const property in business.employees) {
+    business.employees[property].daysOfWeekWorking.push('S');
+    business.employees[property].daysOfWeekWorking.push('Su');
+  }
+}
+
+function addEmployees() {
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://jsonplaceholder.typicode.com/users');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', () => {
+    for (let index = 0; index < 4; index++) {
+      business.employees[xhr.response[index].name] = xhr.response[index];
```

Comment:
You shouldn't be adding the whole response to their `employee` object. Each employee should only have 3 properties. `position`, `daysOfWeekWorking`, & `fullTime`. We don't want all the other data that gets returned from the API